samples/attested_tls/scripts/gen_mrenclave_header.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
Can you please help me understand why this needs to be in a script?

Reply:
That's the only to make bash recognize the containing file is a bash script file
(Each script starts with a "shebang" and the path to the shell that you want the script to use)